tests/lint/test_pylinter.py

```diff
@@ -48,3 +48,11 @@ def test_crash_during_linting(
         assert len(files) == 1
         assert "pylint-crash-20" in str(files[0])
         assert any(m.symbol == "astroid-error" for m in linter.reporter.messages)
+
+
+def test_open_pylinter_denied_modules(linter: PyLinter) -> None:
+    """Test PyLinter open() adds ignored modules to Astroid manager deny list."""
+    MANAGER.module_denylist = {"mod1"}
+    linter.config.ignored_modules = ["mod2", "mod3"]
```

Comment:
Thanks for adding the test coverage.

Would you mind adjusting this test so that it does not leave side effects on the MANAGER constant afterward?

Reply:
Sure; would you rather I store the starting value and restore that, or just restore the default (`set()`)?

Reply:
Default set() is fine, since I don't see us changing that in the future. And if it's anything else, it's the fault of some other hypothetical future test.

Reply:
Done